Makefile

```diff
@@ -104,28 +107,26 @@ docker-tests:
 			exit 1 ;\
 		fi ;\
 	fi ;\
-	tmp_file=$$(mktemp) ;\
+	touch tests_failed ;\
 	if test "x$$FULLSTACK" = x1; then \
-	  perl t/full-stack.t | tee $$tmp_file;\
+	  perl t/full-stack.t && rm tests_failed ;\
 	elif test "x$$SCHEDULER_FULLSTACK" = x1; then \
-		perl t/05-scheduler-full.t | tee $$tmp_file ;\
+	  perl t/05-scheduler-full.t && rm -v tests_failed ;\
 	else \
 	  list= ;\
 	  if test "x$$UITESTS" = x1; then \
 	    list=$$(find ./t/ui -name *.t | sort ) ;\
 	  else \
-	    $(MAKE) checkstyle ;\
+	    $(MAKE) checkstyle || exit 1 ;\
 	    list=$$(find ./t/ -name *.t | grep -v t/ui | sort ) ;\
 	  fi ;\
-          prove ${PROVE_ARGS} -r $$list | tee $$tmp_file ;\
-	fi ;\
-	grep -o -E -v "Looks like you failed [[:digit:]]+ test|not ok [[:digit:]]|Result: FAIL" $$tmp_file ;\
-	test_failed=$$? ;\
-	echo $$test_failed ;\
-	if [[ $$test_failed -ne 0 && $$TRAVIS ]]; then\
-		cat /tmp/openqa-debug.log ;\
-	fi ;\
-	exit $$test_failed
+          prove ${PROVE_ARGS} -r $$list && rm -v tests_failed ;\
+	fi
+	if test ! -r tests_failed; then \
+		sudo mv ${OPENQA_LOGFILE} . ;\
```

Comment:
Shouldn't this be:

> sudo mv ${OPENQA_LOGFILE} /opt/openqa

?

Reply:
Yeah. don't drink and code. This coke is aweful!